src/mainboard/protectli/vault_ehl/romstage.c

```diff
@@ -4,10 +4,53 @@
 #include <types.h>
 #include <soc/meminit.h>
 #include <soc/romstage.h>
+#include <spd_bin.h>
+
+static void mainboard_fill_dq_map_data(void *dq_map_ch0, void *dq_map_ch1)
+{
+	const uint8_t dq_map[2][12] = {
+		{ 0x0F, 0xF0, 0x00, 0xF0, 0x0F, 0xF0,
+		  0x0F, 0x00, 0xFF, 0x00, 0xFF, 0x00 },
+		{ 0x33, 0xCC, 0x00, 0xCC, 0x33, 0xCC,
+		  0x33, 0x00, 0xFF, 0x00, 0xFF, 0x00 } };
+	memcpy(dq_map_ch0, dq_map[0], sizeof(dq_map[0]));
+	memcpy(dq_map_ch1, dq_map[1], sizeof(dq_map[1]));
+}
+
+
+static void mainboard_fill_dqs_map_data(void *dqs_map_ch0, void *dqs_map_ch1)
+{
+	const uint8_t dqs_map[2][8] = {
+		{ 0, 1, 2, 3, 4, 5, 6, 7 },
+		{ 0, 1, 2, 3, 4, 5, 6, 7 } };
+	memcpy(dqs_map_ch0, dqs_map[0], sizeof(dqs_map[0]));
+	memcpy(dqs_map_ch1, dqs_map[1], sizeof(dqs_map[1]));
+}
```

Comment:
These are needed only for LPDDR memory. We do not set them for DIMM modules.
Take a look at CML board what is being done for DDR4 https://github.com/coreboot/coreboot/blob/master/src/mainboard/msi/ms7d25/romstage_fsp_params.c